spec/models/collection_spec.rb

```diff
@@ -4,96 +4,104 @@
 
 RSpec.describe Collection do
   let(:user) { create(:user) }
-  let(:collection) { described_class.create(title: collection_title_fixture, user:) }
+  let(:collection) { create(:collection, title: collection_title_fixture, user:) }
 
   before do
     allow(Notifier).to receive(:publish)
   end
 
-  describe 'Add participants to a collection' do
-    context 'when adding a manager to collection' do
-      it 'only sends a MANAGER_ADDED event' do
-        collection.managers << user
-        expect(Notifier).to have_received(:publish).with(Notifier::MANAGER_ADDED, collection:,
-                                                                                  user:)
-        expect(Notifier).not_to have_received(:publish).with(Notifier::MANAGER_REMOVED)
```

Comment:
Instead of checking if this is not received, change the expectation below that `:published` is ONLY received once.